spec/spec_helper.rb

```diff
@@ -1,26 +1,33 @@
+# frozen_string_literal: true
+
 require "bundler/setup"
 require "sequel"
 require "sequel/extensions/migration"
-require "sequel-bulk-audit"
+require "sequel_bulk_audit"
```

Comment:
Does this work automatically when I add gem to the Gemfile? Or maybe we should add `sequel-bulk-audit.rb` which  requires `sequel_bulk_audit.rb`?

Reply:
It doesn't seem to have any negative effects. I wasn't sure at first when rubocop demanded filename change. But it bundles without problems and works just fine.

Reply:
It will broke the gem code auto loading mechanism of projects those already use this gem.  

just disable an appropriated rubocop rule.

```yaml
Naming/FileName:
  Exclude:
    - lib/sequel-builk-audit.rb
```

Reply:
Allright.Changed filename back, added rule exclusion for it.